content/tutorials/manage-account-settings/use-tickets.md

```diff
@@ -0,0 +1,372 @@
+
+For this tutorial, you can connect directly from your browser by pressing the following button:
+
+{{ start_step("Connect") }}
+<button id="connect-button" class="btn btn-primary">Connect to Devnet</button> <!-- TODO: Testnet -->
+<div>
+  <strong>Connection status:</strong>
+  <span id="connection-status">Not connected</span>
+  <div id='loader-connect' style="display: none;"><img class='throbber' src="assets/img/xrp-loader-96.png"></div>
+</div>
+{{ end_step() }}
+
+
+### {{n.next()}}. Check Sequence Number
+
+(If you already have at least one Ticket available in the ledger, you can skip this step.) You need to get ready to create some Tickets for your other transactions to use.
+
+Before you create any Tickets, you should check what [Sequence Number][] your account is at. You want the current Sequence number for the next step, and the Ticket Sequence numbers it sets aside start from this number.
+
+```js
+async function get_sequence() {
+  const account_info = await api.request("account_info", {
+      "account": "rPT1Sjq2YGrBMTttX4GZHjKu9dyfzbpAYe"
+  })
+  console.log("Current sequence:", account_info.account_data.Sequence)
+  return account_info.account_data.Sequence
+}
+
+let current_sequence = get_sequence()
+```
+
+{{ start_step("Check Sequence") }}
+<button id="check-sequence" class="btn btn-primary connection-required"
+  title="Complete all previous steps first" disabled>Check Sequence Number</button>
+<div id="check-sequence-output"></div>
+{{ end_step() }}
+
+
+
+### {{n.next()}}. Prepare and Sign TicketCreate
+
+(If you already have at least one Ticket available in the ledger, you can skip this step.) You need to prepare the transaction that will create some Tickets.
+
+Construct a [TicketCreate transaction][] using the sequence number you determined in the previous step. Use the `TicketCount` field to specify how many Tickets to create. For example, this example [prepares a transaction](rippleapi-reference.html#preparetransaction) that would make 10 Tickets:
+
+```js
+let prepared = await api.prepareTransaction({
+  "TransactionType": "TicketCreate",
+  "Account": "rPT1Sjq2YGrBMTttX4GZHjKu9dyfzbpAYe",
+  "TicketCount": 10,
+  "Sequence": current_sequence
+})
+console.log("Prepared transaction:", prepared.txJSON)
+let max_ledger = prepared.instructions.maxLedgerVersion
+
+let signed = api.sign(prepared.txJSON, "s████████████████████████████")
+console.log("Transaction hash:", signed.id)
+
+let tx_blob = signed.signedTransaction
+```
+
+Take note of the transaction's hash and `LastLedgerSequence` value so you can [be sure whether or not it got validated](reliable-transaction-submission.html) later.
```

Comment:
I don't like using "Note" as a verb to start a sentence since that feels too similar to "Note: somethingsomething".

Changed to "Record the..." instead